src/openshift/component.ts

```diff
@@ -448,10 +448,8 @@ export class Component extends OpenShiftItem {
 
         if (!componentTypeVersion) return null;
 
-        window.showInformationMessage('Do you want to clone git repository for created Component?', 'Yes', 'No').then((value) => {
-            value === 'Yes' && commands.executeCommand('git.clone', repoURI);
-        });
-
+        const response = await window.showInformationMessage('Do you want to clone git repository for created Component?', 'Yes', 'No');
+        if (response) await commands.executeCommand('git.clone', repoURI);
```

Comment:
what if the response is `No` ?

Reply:
Should we return null if the response is no or we should create the component without cloning the component that's what is was happening before?

Reply:
check for Yes condition and then clone the repo. Cloning the repo is an optional stuff provided to the user. If the response is NO, skip and proceed to component creation.